### PR TITLE
[webnn] Verify outputs shape

### PR DESCRIPTION
**Pull request by BruceDai**
@fdwr @Honry @lisa0314 PTAL, thanks.

### CODE REVIEW

File: webnn/resources/utils.js [hunk 1]
```diff
@@ -67,21 +67,21 @@ const loadTests = (operationName) => {
 };
 
 /**
- * Get exptected data and data type from given resources with output name.
+ * Get exptected sub output resources from given resources with output name.
```
Comment (fdwr) [inline webnn/resources/utils.js:70]:
```suggestion
 * Get expected suboutput resources from given resources with output name.
```

Comment (BruceDai) [reply webnn/resources/utils.js:70]: >  "Suboutput" to me implies a subspan of one of the outputs (a range of sliced elements), whereas this appears to just be grabbing one of the outputs.

That's correct. Sorry for missing your comment, will submit commit to update it.

Comment (BruceDai) [reply webnn/resources/utils.js:70]: Yes，I'm working to fix it.

Comment (BruceDai) [reply webnn/resources/utils.js:70]: How about close it for not trouble others.
And new one pr.

Comment (fdwr) [reply webnn/resources/utils.js:70]: Okie dokie. Well, I'll look at it Monday (just briefly checked my GitHub notifications before bed).

**Comment by BruceDai** [comment]
Sorry for bad merge. Let's close it.

**Pull request closed without merging**